bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -54,7 +59,16 @@ spec:
             fi
           done
           echo
-          exec hyperkube kube-apiserver --openshift-config=/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml --advertise-address=${HOST_IP}
+          trap 'sig=$(($? - 128)); echo "Received signal $(kill -l ${sig})"; touch "${TERM_FILE}"; kill -TERM ${PID}' TERM INT
```

Comment:
do we want to make it numeric with list from 1..64 to make sure we aren't loosing signals? not sure if there is easier way in bash to forward all